libnetwork/netutils/utils.go

```diff
@@ -124,6 +124,9 @@ func GenerateMACFromIP(ip net.IP) net.HardwareAddr {
 // GenerateRandomName returns a new name joined with a prefix.  This size
 // specified is used to truncate the randomly generated value
 func GenerateRandomName(prefix string, size int) (string, error) {
+	if size > 64 {
+		panic("size cannot be greater than 64")
```

Comment:
`panic` is generally an anti-pattern except in exceptional cases.  This function already returns `(string, error)`, so you can do this instead:
```suggestion
		return "", fmt.Errorf("name: size should be less than 64 but was %d", size)
```

Reply:
Good suggestion! I have to check if 64 is a valid value for `size` (I don't think it is). I prematurely pushed my branch so I'll do more testing and push the updated changes.

Reply:
64 is a valid size. I updated your suggestion by saying `less than or equal to 64`. It takes longer to grok, which is why I originally wrote `greater than 64`. Which message is preferred?

Reply:
I think either phrasing is fine.  Thanks for updating!